src/player.cpp

```diff
@@ -129,11 +129,16 @@ void add_action(player& p, action_type action) noexcept
 
 bool are_colliding(const player &lhs, const player &rhs) noexcept
 {
+  if (lhs.get_state() == player_state::active &&
+      rhs.get_state() == player_state::active)
+  {
```

Comment:
In C++ it is common to do

```c++
if (a == b
  && c == d
  && e == f
)
{
  // Stuff
}
```

Notice the place for the `&&`: it's at the **start** at each line (note in R one cannot do this)

Reply:
did it!